### DISCUSSION TITLE
Question for howto install age inside Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by cesarjorgemartinez** [FiloSottile/age#524] [Q&A]
Hi,

At one machine I have Ubuntu 20.04.
Then the age has not package (I think)

Then I installed as (coping to /usr/local/bin):
On Windows, Linux, macOS, and FreeBSD you can use the pre-built binaries.

https://dl.filippo.io/age/latest?for=linux/amd64
https://dl.filippo.io/age/v1.1.1?for=darwin/arm64

Exists other form to use packages in Ubuntu 20.04?

I see this:
Debian 11+ (Bullseye) | apt install age/bullseye-backports             (enable backports for age v1.0.0+)

Its compatible and same instructions?
https://backports.debian.org/Instructions/#index2h2

Or can be exists other forms to use apt-get install?

Regards,
Cesar Jorge

**Comment by FiloSottile** [comment]
Currently, there is no backport for Ubuntu 20.04, so the pre-built binary is the recommended way to install.
